packages/gluestick-cli/cli.js

```diff
@@ -66,27 +68,68 @@ commander
     } catch (error) {
       exitWithError('Invalid package.json');
     }
-    const wmlBin = path.join(__dirname, './node_modules/.bin/wml');
-    spawn.sync(
-      wmlBin,
-      ['rm', 'all'],
-      { stdio: 'inherit' },
-    );
-    spawn.sync(
-      wmlBin,
-      ['add', srcPath, path.join(process.cwd(), 'node_modules/gluestick')],
-      { stdio: 'inherit' },
-    );
-    spawn.sync(
-      'watchman',
-      ['watch', srcPath],
-      { stdio: 'inherit' },
-    );
-    spawn.sync(
-      wmlBin,
-      ['start'],
-      { stdio: 'inherit' },
-    );
+    const convertFilePath = filePath => filePath.replace(srcPath, path.join(process.cwd(), 'node_modules/gluestick'));
+    const watcher = chokidar.watch(`${srcPath}/**/*`, {
+      ignored: [
+        /(^|[/\\])\../,
+        /.*node_modules.*/,
+      ],
+      persistent: true,
+    });
+    watcher
+      .on('ready', () => {
+        console.log(chalk.blue('Watching for changes...'));
+        watcher.on('add', filePath => {
+          const destPath = convertFilePath(filePath);
+          fs.copy(filePath, destPath, (err) => {
+            if (err) {
+              console.error(chalk.red(err));
+            } else {
+              console.log((chalk.green(`${filePath} -> ${destPath} [added]`)));
+            }
+          });
+        });
+        watcher.on('change', filePath => {
```

Comment:
This is almost identical to `add` event handler, so this should be extracted to separate function - see above.